Source/WebCore/platform/graphics/DestinationColorSpace.cpp

```diff
@@ -86,6 +86,11 @@ const DestinationColorSpace& DestinationColorSpace::DisplayP3()
 }
 #endif
 
+DestinationColorSpace::DestinationColorSpace()
```

Comment:
this is unfortunate and you could side-step this for now by just not having the get... function.

Reply:
I believe this is also needed to receive the `Set...` message in the GPU process, the generated code does `DestinationColorSpace returnValue = { };`, so we need this default constructor there.

I think you were suggesting above to maybe use another hand-crafted enum, but we'd have to use that enum in all the `GraphicsContextGL` classes, right? It seemed better for me to try and use the real `DestinationColorSpace`.

An alternative would be to modify generate-gpu-webgl, so that instead of doing `X returnValue = { };`...`returnValue = mContext->doX();`, it would instead generate one line `X returnValue = mContext->doX();` (but that program looks quite complex and scary already!)